src/components/NavigationBar.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Nav, Navbar, NavDropdown } from 'react-bootstrap';
+import { Link, NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHome } from '@fortawesome/free-solid-svg-icons';
+import { faHtml5, faCss3, faJs } from '@fortawesome/free-brands-svg-icons';
+
+const NavigationBar = () => {
+  return (
+    <Navbar collapseOnSelect expand="lg" bg="primary" variant="dark">
```

Comment:
I think something like `sm` for `expand` would be better. On a smaller screen the links can still fit in the header so we don't need to change to show the toggle menu until we have a much smaller screen

Reply:
Done!